src/rpc/multipass.proto

```diff
@@ -280,7 +280,7 @@ message StartRequest {
 
 message StartReply {
     string log_line = 1;
-    string start_message = 2;
+    string reply_message = 2;
```

Comment:
@gerboland,

No, I don't think it's absolutely necessary, but I did this to make it more generic for when setting the reply message in the following piece if code:
https://github.com/CanonicalLtd/multipass/blob/a47a4e969a36dcbe9d6796c650a0e821bae8c1f9/src/daemon/daemon.cpp#L2142-L2143
`Reply` here is templated, but if left the RPC the way it was, it would be something like `reply.set_start_message(blah)` which didn't seem clean to me.

Reply:
Fair enough!